src/redux/middlewares/loginSessionMiddleware.ts

```diff
@@ -14,38 +14,41 @@ const throttledSetNewExpires = throttle(() => {
   setLoginExpiresAt(getNewLoginExpiresTimestamp());
 }, 5000);
 
-export const loginSessionMiddleware: any = (store: StoreType) => (
-  next: (action: PayloadAction) => void
-) => (action: PayloadAction) => {
-  if (whitelistedActions.includes(action.type)) {
+export const loginSessionMiddleware: any =
```

Comment:
better avoid using any if possible

Reply:
It's not my change. Just eslint reformat code on this file. I am not sure the reason of using `any` here